site/_includes/partials/chrome-100-hero-section.njk

```diff
@@ -0,0 +1,48 @@
+{% block content %}
+<section id="back-to-top" class="hero-section">
+  <share-button message="#100CoolWebMoments brought back some memories! Loved going through this timeline of the web. Check it out at https://developer.chrome.com/100">
+    <a class="share-button__link gap-right-200 mobile-share-logo"
+        href="https://twitter.com/share?url={{site.url}}{{ page.url | stripDefaultLocale}}&text=##100CoolWebMoments brought back some memories! Loved going through this timeline of the web. Check it out at https://developer.chrome.com/100"
+        target="_blank" data-label="twitter share" data-category="share" data-action="click">
+      <svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
```

Comment:
The same icon is used here: https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_includes/partials/breadcrumbs.njk

Please extract into _includes/icons and reuse from there.

Reply:
> For the Chrome DevTools card in year 2009, change the description to: "Open-source and built upon Webkit's Inspector."

Done.

Reply:
> The same icon is used here: https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_includes/partials/breadcrumbs.njk
> 
> Please extract into _includes/icons and reuse from there.

Done

Reply:
> The same icon is used here: https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_includes/partials/breadcrumbs.njk
> 
> Please extract into _includes/icons and reuse from there.

Done